### ISSUE TITLE
Add a control to open a file in native mode explicitly

### ISSUE THREAD

**Issue by OleksiyRudenko** [brumm/tako#3]
@brumm great tool with a UX that feels natural. Kudos!

At v.0.0.3 in order to open a file in "traditional" mode to be able to edit it employing GitHub WebUI one has to right/middle-click a file/directory entry and open the file/directory in a different tab from the context menu.

It would be great to have an explicit control to open a file/directory in native mode.

One of the approaches might be adding a control (e.g. an icon with a tooltip "Open in a new tab") to each entry. Possible issue: when a file preview is open the control may become obscured by the preview pane as the file listing "sidebar" is of fixed width.

Another approach might be to add a sticky top bar in the preview pane with a link to open in a new tab. Bonus: add other controls there, like "close preview". Possible issue: controls to open directories must be somewhere else for preview pane doesn't fire on directory entry click.

**Comment by brumm** [comment]
Great ideas! I think I'm going to start with some hover-only icon (or maybe even use the file icon itself)... Thee top bar of the preview pane is definitely coming at some point, I was planning to offer some syntax theme/font size options there :)

**Comment by madhavarshney** [comment]
@brumm This is a really, really nice tool! Here is something I was expecting:

Another type of click that should be detected in addition to middle clicks is `cmd + click` on macOS and `ctrl + click` on other platforms. All major browsers (as far as I know) support opening a link in a new tab when you `ctrl`/`cmd` + `click` them. I believe the following would be changed to something like:

https://github.com/brumm/tako/blob/e981f7845960def9fd6ead2870518742be32bd65/src/components/Node.js#L129-L131

```js
  const macosCmd = window.navigator.platform.indexOf('Mac') > -1
                   && event.nativeEvent.metaKey;

  if (!macosCmd && !event.nativeEvent.ctrlKey && event.nativeEvent.which !== 2) {
    event.preventDefault();
  }
```

**Comment by brumm** [comment]
Fixed by #30
